Develop/assets/css/style.css

```diff
@@ -85,6 +94,12 @@ code {
 .card-column {
   /* when the screen resizes, make it so our columns don't jump in size */
   transition: flex-basis 500ms linear;
+
+  /* new */
+  display: flex;
+  flex-flow: row wrap;
+  /* flex: 0 1 100%;
+  I think this may not do anything */
 }
```

Comment:
Now applied to the main element. Makes it a flex container with row direction and wrap applied to its items - the code cards.